### DISCUSSION TITLE
Incoming follow requests for private accounts

### DISCUSSION THREAD

**Discussion by markpatx** [sbertix/Swiftagram#249] [Q&A] [answered]
Can swiftagram be used to retrieve a list of incoming follow request for a private account from https://www.instagram.com/accounts/activity/?followRequests=1 and confirm/reject them?

**Comment by sbertix** [comment, marked as answer]
Hey @markpatk,

Sure. You can get all following requests for your logged in accounts with `Endpoint.users.requests`, accept them with `Endpoint.user(/* the id for the user who sent you the request */).request.approve()` and decline them with `Endpoint.user(/* the id for the user who sent you the request */).request.decline()`.
